### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2490: ordinal not in range(128)

### ISSUE THREAD

**Issue by Jeiyoon** [chrisjbryant/errant#32]
Hi 😊

I've encountered a problem while using errant:

I think there are conflicts between the version of python and spacy and I couldn't fix it 

~~~
Python 3.6.13 |Anaconda, Inc.| (default, Jun  4 2021, 14:25:59)
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import errant
>>> errant.load('en')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/anaconda3/envs/errant200/lib/python3.6/site-packages/errant/__init__.py", line 19, in load
    classifier = import_module("errant.%s.classifier" % lang)
  File "/root/anaconda3/envs/errant200/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/anaconda3/envs/errant200/lib/python3.6/site-packages/errant/en/classifier.py", line 40, in <module>
~~~

I use 

- Ubuntu 20.04
- python 3.6.13
- spacy 1.9.0

Thank you

**Comment by chrisjbryant** [comment]
Hiya!

Sorry for the delay - I'm travelling at the moment.
I've not seen that error before, but can I ask what version of ERRANT you installed? With the setup you're using, you need to install `errant==2.1.0` as later versions won't work with spacy 1.9.0.

**Comment by Jeiyoon** [comment]
> 

problem solved! I installed 2.1.0 version

i hope you had a great trip

Thanks ☺️
